discovery/short_interest_api.py

```diff
@@ -1,106 +1,117 @@
     df_high_short_interest = pd.DataFrame(columns=a_high_short_interest_header)
     df_high_short_interest.loc[0] = ['', '', '', '', '', '', '']
 
-    a_high_short_interested_stocks = re.sub('<!--.*?//-->','', text_soup_high_short_interested_stocks.find_all('td')[3].text, flags=re.DOTALL).split('\n')[2:]
-    a_high_short_interested_stocks[0] = a_high_short_interested_stocks[0].replace('TickerCompanyExchangeShortIntFloatOutstdIndustry','')
+    a_high_short_interested_stocks = re.sub('<!--.*?//-->', '',
+                                            text_soup_high_short_interested_stocks.find_all('td')[3].text,
+                                            flags=re.DOTALL).split('\n')[2:]
+    a_high_short_interested_stocks[0] = a_high_short_interested_stocks[0].replace(
+        'TickerCompanyExchangeShortIntFloatOutstdIndustry', '')
 
     l_stock_info = list()
     for elem in a_high_short_interested_stocks:
-        if elem is '':
+        if elem == '':
             continue
-            
+
         l_stock_info.append(elem)
-            
+
         if len(l_stock_info) == 7:
             df_high_short_interest.loc[len(df_high_short_interest.index)] = l_stock_info
             l_stock_info = list()
-        
+
     pd.set_option('display.max_colwidth', -1)
     print(df_high_short_interest.head(n=ns_parser.n_num).to_string(index=False))
     print("")
 
 
 # ---------------------------------------------------- LOW_FLOAT -----------------------------------------------------
 def low_float(l_args):
-    parser = argparse.ArgumentParser(prog='low_float', 
-                                     description='''Print top stocks with lowest float. LowFloat.com provides a convenient 
-                                     sorted database of stocks which have a float of under 10 million shares. Additional key 
-                                     data such as the number of outstanding shares, short interest, and company industry is 
-                                     displayed. Data is presented for the Nasdaq Stock Market, the New York Stock Exchange, 
-                                     the American Stock Exchange, and the Over the Counter Bulletin Board. 
+    parser = argparse.ArgumentParser(prog='low_float',
+                                     description='''Print top stocks with lowest float. LowFloat.com provides a
+                                     convenient
+                                     sorted database of stocks which have a float of under 10 million shares.
+                                     Additional key
+                                     data such as the number of outstanding shares, short interest, and company
+                                     industry is
+                                     displayed. Data is presented for the Nasdaq Stock Market, the New York Stock
+                                     Exchange,
```

Comment:
This text looks a bit all over the place. Would be nice for it to be easily readable in the code